src/Microsoft.Azure.WebJobs.Host/Bindings/SysBindingData.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Host.Bindings.Path;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    /// <summary>
+    /// Built in object for all binding expressions. 
+    /// </summary>
+    /// <remarks>
+    /// It's expected this class is created and added to the binding data. 
+    /// </remarks>
+    internal class SysBindingData
+    {
+        // The name for this binding in the binding expressions. 
+        public const string Name = "sys";
+
+        // A name that can't be overwritten by the user that we cna place in the binding data dictionary
+        // and use to unambiguously retrieve this later. 
+        private const string InternalName = "$sys";
```

Comment:
The problem is that we need a unique name that can't conflict with user binding data (so that it doesn't get overwritten. I used '$' char for that. . 

---
In reply to: [116879586](https://github.com/Azure/azure-webjobs-sdk/pull/1148#discussion_r116879586) [](ancestors = 116879586)